composer.json

```
@@ -4,11 +4,11 @@
         "masnathan/odtphp": "dev-teclib"
     },
     "require-dev": {
-        "glpi-project/tools": "^0.7",
+        "glpi-project/tools": "^0.7.2",
```

It is preferable to not explicitely set the `bugfix` number in the required version.
```suggestion
        "glpi-project/tools": "^0.7",
```